Taskfile.dist.yml

```diff
@@ -1,6 +1,8 @@
 version: '3'
 
 vars:
+  PWD:
+    sh: pwd
```

Comment:
Is `pwd` a command on Windows? Can you add a comment why this change was added.

Reply:
`pwd` is an alias in Windows PowerShell for `Get-Location`. I've verified it works on Windows when using Task. And sure I can add a comment.